.github/workflows/frontend-deploy-staging.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy Frontend to Staging
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
For workflow dispatch it would be nice to be able to choose the target network, so that this can also be used to deploy to the mainnet test canisters test_fe_1..4  But this can be done in another PR.

Reply:
Good one!